get-started/learn-fuchsia.md

```diff
@@ -1,31 +1,47 @@
+<!--
 # Get started
-
+-->
+# 开始步骤
+<!--
 Fuchsia is a modern open source operating system that’s simple, secure,
 updatable, and performant. It provides core operating system functions like
 system resource management, a driver framework, and software abstractions.
 Fuchsia is a general purpose operating system designed to power a diverse
 ecosystem of hardware and software.
-
+-->
+Fuchsia 是一个现代化开源操作系统，它简单、安全、可更新以及运行效率高。同时提供了操作系统的核心功能，比如系统资源管理、驱动框架以及软件抽象。Fuchsia 是一个支持常见设备的操作系统，被设计于助力软硬件多样的生态系统。
+<!--
 ## SDK or source tree?
-
+-->
+## SDK 还是源代码?
+<!--
 Fuchsia is an open source project in active development at Google. There are
 two ways to work with Fuchsia: you can build products and software for Fuchsia
 using the SDK, or contribute to the source tree to build the Fuchsia platform.
-
+-->
+Fuchsia 是 Google 积极开发的一个开源项目。现在有两种方式来参与 Fuchsia：可以使用 Fuchsia 的 SDK 来为 Fuchsia 开发产品和软件，或者为 Fuchsia 平台的源代码贡献一份力。
+<!--
 ## SDK - Build software for Fuchsia
-
+-->
+## SDK - 为 Fuchsia 开发软件
+<!--
 Fuchsia has an SDK with tools to help you get started developing
 software that will run on Fuchsia. Before you begin building your software, it
 is recommended you learn about the fundamentals of using the Fuchsia SDK.
+-->
+Fuchsia 的 SDK 配套有一些工具可以帮助您为 Fuchsia 开发软件。在您开始开发软件之前，建议您学习一些 Fuchsia SDK 的基本使用方法。
 
 <a class="button button-primary"
-    href="/get-started/sdk/learn">Fundamentals for SDK</a>
-
+    href="/get-started/sdk/learn">SDK 基础</a>
+<!--
 ## Source tree - Contribute to Fuchsia
-
+-->
+## 源代码 - 为 Fuchsia 贡献
+<!--
 As Fuchsia is an open source project, you can contribute to the Fuchsia source tree. Before you
 begin contributing, it’s recommended that you learn about the fundamentals about
 the Fuchsia platform to understand how Fuchsia works.
-
+-->
+Fuchsia 作为一个开源项目，您当然可以为它的源代码做贡献。在您开始编写源码之前，建议您学习一些 Fuchsia 的基础以更好的理解它的运行原理。
```

Comment:
```suggestion
由于 Fuchsia 是一个开源项目，因而您可以为 Fuchsia 源代码树贡献代码。在您开始贡献代码之前，建议您了解关于 Fuchsia 平台的基础知识，以便理解 Fuchsia 的工作原理。
```

as 做连词，这里是“因为/由于”的意思，类似于 because。

learn about

how ... work(s) 翻译成“工作原理”你觉得如何？比“运行”更宽泛一些。